openchemistry/__init__.py

```diff
@@ -436,23 +435,37 @@ def load_orbital(self, mo):
     def url(self):
         return '%s/calculations/%s' % (app_base_url.rstrip('/'), self._id)
 
+class AvogadroProvider(CjsonProvider):
+    def __init__(self, molecule):
+        self._molecule = molecule
+        self._cjson = None
+
+    @property
+    def cjson(self):
+        if self._cjson is None:
+            conv = avogadro.io.FileFormatManager()
+            cjson_str = conv.write_string(self._molecule, 'cjson')
+            self._cjson = json.loads(cjson_str)
+        return self._cjson
+
 class Visualization(ABC):
     def __init__(self, provider):
         self._provider = provider
         self._params = {}
 
     @abstractmethod
-    def show(self, viewer='moljs', spectrum=False, volume=False, isosurface=False, menu=True, mo=None, isovalue=None, mode=-1, play=False, alt=None):
+    def show(self, viewer='moljs', spectrum=False, volume=False, isosurface=False, menu=True, mo=None, iso=None, transfer_function=None, mode=-1, play=False, alt=None):
```

Comment:
isovalue=>iso is a breaking change?

Reply:
It's not, calling it `isovalue` in the first place was a breaking change, this reverts that

Reply:
ok far enough